src/utils.js

```diff
@@ -0,0 +1,40 @@
+function sendJsonResponse(res, statusCode, data) {
+  res.writeHead(statusCode, { 'Content-Type': 'application/json' });
+  res.end(JSON.stringify(data));
+}
+
+function respondWithError(res, errors) {
```

Comment:
Is your error status-code always going to be 400?
 What if it's an internal server error? Or any derivation of 400 (401, 402...) ?

Reply:
@mfcastilho